src/pages/experimental/delete-pattern/index.mdx

```diff
@@ -1,94 +1,213 @@
+## Delete Low impact
 
-A medium-impact deletion is one that cannot be reversed, but would not be catastrophic if done accidentally. When deleting is medium-impact, a confirmation dialog should be presented to the user which displays:
+Requiring the user to confirm deletion is generally recommended. However, in very low impact situations, such as when an ‘undo’ option is available, user confirmation may not be required.
 
-- The name of the resource
-- Consequences of the deletion
-- The action cannot be undone
+<Row>
+<Column colLg={8}>
+
+![Example of delete options in context without warnings for low impact delete actions](images/LowImpactDelete.png)
 
-Optionally, when the delete button is selected it changes to `Deleting…`.
+<Caption> Low impact delete options</Caption>
+</Column>
+</Row>
+
+## Remove Medium impact
+
+Removing has a medium impact when the action can't be reversed, but would not be catastrophic if done accidentally.
+
+Text should tell the user the consequences of the removal and that the action cannot be undone.
 
 <Row>
 <Column colLg={8}>
 
-![Example of a standard delete modal](images/6.png)
+![Example of a medium impact remove modal](images/first-view.png)
+
+<Caption>
+  {' '}
+  Removal modals should include consequences of removal, and "This action cannot
+  be undone" statement.
+</Caption>
+
+</Column>
+</Row>
+
+## Remove Low impact
+
+As removing does not destroy an asset, typically user confirmation is not required.
+
+<Row>
+<Column colLg={8}>
 
-![Example of a standard delete modal in its "deleting" state](images/1_1.png)
+![Example of remove options in context without warnings for low impact remove actions.](images/LowImpactRemove.png)
 
-![Example of a standard delete modal in its "deleting" state](images/1_2.png)
+<Caption> Example low impact remove options</Caption>
 
 </Column>
 </Row>
 
-You can choose to show a notification confirming deletion is completed. This is useful when the deletion of the resource takes more than a few moments.
+## Anatomy
+
+### Success modal
+
+In high or medium impact flows, an optional final dialog can be used to signify success at the end of a remove or delete action.
+
+<Row>
+<Column colLg={8}>
+
+![Example of a delete successful modal](images/DoneSerious.png)
+
+<Caption> Use a success modal to show that deleting is complete.</Caption>
+
+![Example of a remove successful modal](images/DoneMedium.png)
+
+<Caption> Use a success modal to show that removing is complete.</Caption>
+
+</Column>
+</Row>
+
+### Notification
+
+Use an optional notification to confirm a delete or remove action is completed. This is useful when the action takes more than a few moments.
 
 <Row>
 <Column colLg={8}>
 
 ![Example of a successful delete notification](images/7_updated.png)
 
+<Caption>
+  {' '}
+  An optional notification can be used to confirm the delete or remove action
+  with medium or high impact actions.
+</Caption>
+
 </Column>
 </Row>
 
-## Low impact
+### Requested modal
```

Comment:
I would remove this whole section (as my previous suggestion is to make the success modal section an optional passive modal section which would encompass this)

Reply:
Will also follow this one up